src/pages/api/auth/[...nextauth].js

```diff
@@ -92,8 +144,13 @@ const options = {
 
     // see: https://next-auth.js.org/configuration/callbacks#jwt-callback
     async jwt({ token, user, account }) {
+      // when sign in with credentials return user
+      if (account?.type === "credentials") {
+        return user;
+      }
```

Comment:
`jwt` callback should return `token`. Why are we return `user`?

Reply:
the [JWT](https://next-auth.js.org/v3/configuration/callbacks#jwt-callback) returns the object which will be encoded into the token for the frontend.  In our case, the user object returned by the ~Credentials` Provider fits that. The token and account on the other hand are null in this case @kilemensi